cla_backend/libs/eligibility_calculator/calculator.py

```diff
@@ -87,14 +91,11 @@ def _do_cfe_civil_check(self):
 
     @staticmethod
     def _is_data_complete_enough_to_call_cfe(case_data):
-        if not hasattr(case_data, "facts"):
-            # facts section is at the root of everything, so we require it call CFE
-            return False
         if EligibilityChecker._under_18_passported(case_data):
             # no more info needed
             return True
-        if not hasattr(case_data.facts, "dependants_young") and not (
-            hasattr(case_data.facts, "on_passported_benefits") and case_data.facts.on_passported_benefits
+        if (case_data.facts.on_passported_benefits is None) or (
```

Comment:
This PR changes the outcome for the edge case that `on_passported_benefits=None and dependants_young != None` - this PR changes it to say it can't call CFE. Previously it could call CFE, assuming not on passported benefits, and that seems fine to me - or did you spot a problem with that? If it stays, then we need to explain that situation in the comment and log comment.